### DISCUSSION TITLE
Needing help contributing to an older release v7.x

### DISCUSSION THREAD

**Discussion by kinueng** [eslint/eslint#15095] [General] [answered]
Hello,

I would like to contribute a `Chore` pull request to upgrade `strip-ansi` to [v6.0.1](https://github.com/chalk/strip-ansi/releases/tag/v6.0.1) so ESlint 7.x can pickup a fix for [CVE-2021-3807](https://nvd.nist.gov/vuln/detail/CVE-2021-3807) found in ansi-regex v5.0.1.

Output from `npm list ansi-regex` on eslint repo
```
└─┬ eslint@7.32.0
  └─┬ strip-ansi@6.0.0
    └── ansi-regex@5.0.0 <--- need up upgrade this to 5.0.1
```

I don't see a 7.x branch to target my pull request.  Is there a process for backporting fixes to older eslint releases?

**Comment by aladdin-add** [comment, marked as answer]
no, we do not maintain multi release line (e.g. 7.x).

btw, it does not block upgrading to `strip-ansi$v6.0.1`, as it was using `^`.  Just try:
1. `npm audit fix` (if using npm)
2. or removing your lockfile and reinstall.
